### PR TITLE
Capstone Project

### PR DESCRIPTION
**Pull request by ismailakinkunmi**
build home and about pages,
Each of these pages should have versions for 2 different screen sizes mobile and desktop,
The home page should have a link in the menu to the about page,

### CODE REVIEW

File: css/components/_past.scss [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+      & .content {
+        border-radius: 5px;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        background-color: transparentize($primary, 0.2);
+        width: 100%;
+        height: 100%;
```
Comment (lily-coder) [inline css/components/_past.scss:52]: - [ ] [REQUIRED] As stated in the project requirements, 
### The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable.
Please fix your mobile design of the Past Youth Weeks section; that the transparent orange cover may cover the photos fully to match the original design.
### Your Design
twa1

### Original Design
twa2

File: css/components/_header.scss [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+      padding: $basic-padding * 3;
+      width: 70%;
+    }
+
+    & > h1 {
+      font-family: $header-font;
+      font-size: 50px;
+      background-image: url(../../images/BlobGroup.svg);
+      background-size: cover;
+      // background-clip: text;
```
Comment (lily-coder) [inline css/components/_header.scss]: - [ ] [REQUIRED] Please get rid of this and all other commented unwanted code as it is a good practice to make your code clean and DRY.😊

File: README.md [hunk 3]
```diff
@@ -1 +1,35 @@
...
+
+1. Name: Adekunle Akinkunmi Ismail
+2. Github: @<https://github.com/ismailakinkunmi>
+3. LinkedIn: @<https://www.linkedin.com/in/ismail-akinkunmi-adekunle-b810aa155/>
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
```
Comment (lily-coder) [inline README.md:29]: - [ ] [REQUIRED] As stated in the project requirements
The [Creative Commons license](https://www.behance.net/adagio07) of the design requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project. Please fix that and submit it for review.😊

File: README.md [hunk 4]
```diff
@@ -1 +1,35 @@
...
+
+## Built With
+
+- HTML
+- SCSS
+- JavaScript
+
+## Live Project
+
+[Live Link](http://127.0.0.1:5500/index.html)
```
Comment (lily-coder) [inline README.md]: - [ ] [REQUIRED] Please make use of GitHub Pages to deploy the web pages and test to see that the link is working. Your current link can only work on your local machine, but making use of Github pages will enable anyone across the network to view your work without cloning the repo to their local machines.😊

**Review by lily-coder — CHANGES_REQUESTED** [review]
Hi @ismailakinkunmi , 

Good job so far!
To Highlight! 🟢
✔️ Great UI Design
✔️ Good commit messages
✔️ Correct Github flow was used
✔️ Great use of transitions


There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
image

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [REQUIRED] As stated in the project requirements,
### Once you have finished the development of the project, you should record a video presenting the features of the project you built and add it to the pull request description. It is a video with a maximum length of 5 minutes. The content of the video should include:

- A description of the project.
- A demo of the project features (different pages and different screen sizes).
- You should also highlight some interesting piece of code or something you built that you are very proud of.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: css/components/_past.scss (hunk 1 above)
Comment (ismailakinkunmi) [reply css/components/_past.scss:52]: Screenshot (64)

Comment (ismailakinkunmi) [reply css/components/_past.scss:52]: This is what I
 have here, can you enlight me more on it

Comment (MrBrN197) [reply css/components/_past.scss:52]: @ismailakinkunmi they are referring to your mobile version. your desktop version is fine.

File: about.html [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+        </div>
+
+        <div class="card">
+          <div class="content">
+            <h4>2013</h4>
+            <p>YouthWeek in london</p>
+          </div>
+        </div>
+      </div>
+    </section>
```
Comment (MrBrN197) [inline about.html:101]: - [ ] As the previous reviewer stated please make this design match the reference design. your design matches in the desktop version. but in the mobile version it looks like this
## Your mobile version
image
## Reference Design
image

File: README.md [hunk 6]
```diff
@@ -1 +1,36 @@
...
+
+## Built With
+
+- HTML
+- SCSS
+- JavaScript
+
+## Live Project
+
+This site was built using [GitHub Pages](https://ismailakinkunmi.github.io/Capstone1/)
```
Comment (MrBrN197) [inline README.md:13]: - [ ] You have corrected the link but unfortunately when I visit your GitHub pages link. your website doesn't show up. please make sure that you are correctly deploying your project to GitHub pages.  I think you should deploy from the branch you are working from to fix this issue. 
## Current Deployed website
image

File: scripts/menu.js [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+
+function toggleMenu() {
+  if (menuBtn.classList.contains('active')) {
+    document.body.style.overflow = 'auto';
+  } else {
+    document.body.style.overflow = 'hidden';
+  }
+  menuBtn.classList.toggle('active');
+  mobileMenu.classList.toggle('show');
+}
```
Comment (MrBrN197) [inline scripts/menu.js:12]: - [ ] [OPTONAL] Please kindly try to always use ES6 syntax.
```suggestion
const toggleMenu = () => {
  if (menuBtn.classList.contains('active')) {
    document.body.style.overflow = 'auto';
  } else {
    document.body.style.overflow = 'hidden';
  }
  menuBtn.classList.toggle('active');
  mobileMenu.classList.toggle('show');
}
```

File: README.md [hunk 8]
```diff
@@ -1 +1,36 @@
...
+
+## Video Presentation
+
+[Loom Link](https://www.loom.com/share/74a5724e4b0e46b299357fd4fd807371/)
```
Comment (MrBrN197) [inline README.md:17]: - [ ] Thank you for adding a video demo. However, you are required to be visible in the video as you explain what you did. 
- [ ] remember to highlight a small piece of code you are proud of.
- [ ] [OPTIONAL] please try to speak a bit loader and more clearly as it is somewhat difficult to hear you explain.

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi, @ismailakinkunmi  👋 

You've done a really nice job so far implementing the changes from the previous reviewer.
There are some issues that you still need to work on to go to the next project but you are almost there!
## To highlight 👏👏
- ✔ Design matches really well

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/components/_past.scss (hunk 1 above)
Comment (ismailakinkunmi) [reply css/components/_past.scss:52]: Screenshot (65)

Comment (ismailakinkunmi) [reply css/components/_past.scss:52]: desktop are side by side while mobile is on top of one another, please make the feedback clear or maybe we should both look into it together.

Comment (MrBrN197) [reply css/components/_past.scss:52]: @ismailakinkunmi Hi, you are showing me the desktop version. we are referring to the **mobile** version. 
Please put your screen on mobile version and look at the red background of the image. the red background doesn't overlay the full image on your mobile version it is visible only on a small piece of the top of the image. 
## Your Design (On Mobile)
image

**Comment by ismailakinkunmi** [comment]
https://www.loom.com/share/836bf949265e489c83223a75e03628aa
Please watch the mobile and desktop versions and note the feedback is not clear to me .

**Comment by ismailakinkunmi** [comment]
you are supposed to work or continue on the previous review, I dropped a
message I don't understand  the feedback, I sent a screenshot for it, you
suppose to work on that one with me rather pointing out another thing, did
you want me to repeat the whole week 6 because I don't understand your
review at all .

**Review by MrBrN197 — APPROVED** [review]
## Hi, @ismailakinkunmi  👋

- ✔ I can see that your Github pages is now working. 
- ✔ We've discussed on a zoom call and I can see that your "Past Youth Weeks" section is working correctly.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**